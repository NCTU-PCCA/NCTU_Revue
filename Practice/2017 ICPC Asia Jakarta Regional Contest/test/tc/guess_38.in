80148019038544151 52
>><><><>>>><><>>>><<<<>>><<>><><<<<<>><<>><<<<><<>><
