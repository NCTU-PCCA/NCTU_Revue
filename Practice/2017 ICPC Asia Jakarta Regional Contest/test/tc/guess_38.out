1 2 80148019038544151 3 80148019038544150 4 80148019038544149 5 6 7 8 80148019038544148 9 80148019038544147 10 11 12 13 80148019038544146 80148019038544145 80148019038544144 80148019038544143 14 15 16 80148019038544142 80148019038544141 17 18 80148019038544140 19 80148019038544139 80148019038544138 80148019038544137 80148019038544136 80148019038544135 20 21 80148019038544134 80148019038544133 22 23 80148019038544132 80148019038544131 80148019038544130 80148019038544129 24 80148019038544128 80148019038544127 25 26 80148019038544126
